use var where apparent

SimonCropp opened the pull request.

JimBobSquarePants commented:
Got a failing test on here.
https://ci.appveyor.com/project/six-labors/imagesharp/builds/26661327/job/sv4nakp1mge7cl3k/tests

https://travis-ci.org/SixLabors/ImageSharp/builds/571259943?utm_source=github_status&utm_medium=notification

JimBobSquarePants commented on tests/ImageSharp.Tests/Colorspaces/RgbTests.cs:
```diff
@@ -49,7 +49,7 @@ public void RgbAndRgb24Operators()
             const byte b = 255;
 
-            Rgb24 rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
+            var rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
```
Bug is in the test itself. We're not implicitly casting anymore.

SimonCropp replied:
will revert that one

SimonCropp commented:
@JimBobSquarePants notes. will have a look

SimonCropp commented:
now the weird thing is, now when i build i get a heap of errors about "dont use var".

so it think there might be something wrong with the editor config

@JimBobSquarePants can u clone and verify?

JimBobSquarePants commented:
@SimonCropp Will do!

JimBobSquarePants commented:
@SimonCropp Not seeing any warnings about that at all. Neither does Appveyor. 

https://ci.appveyor.com/project/six-labors/imagesharp/builds/26661327/job/sv4nakp1mge7cl3k#L486

SimonCropp commented:
@JimBobSquarePants ok i deleted and re-clones. no more errors. i think is must have been my beta of r# (which i moved to stable a few days ago) had messed with my intelisense via some old cached files.

JimBobSquarePants commented:
@SimonCropp that’s great to hear! I’m amazed though you’re able to use R# with this project, it grinds my system to a halt!
